vllm/config.py

```diff
@@ -55,6 +56,9 @@ class ModelConfig:
         max_context_len_to_capture: Maximum context len covered by CUDA graphs.
             When a sequence has context length larger than this, we fall back
             to eager mode.
+        tensorizer_path: File path or s3 URI to the tensorizer model weights
```

Comment:
i'm wondering if maybe it would make sense to have these be a dict or dataclass (`TensorizerConfig`?) of misc TensorDeserializer args. this would facilitate leveraging additional tensorizer functionality like encryption or plaid mode.

Reply:
I like that idea. I imagine this would increase the overhead of `tensorizer` literacy for `vllm` users, if this were to hypothetically not just be used by our team internally? My aim for this was to make it agnostic of `tensorizer`'s library and only need to have tensorized weights findable, but that requires tensorizing weights in the first place, so some understanding of how `tensorizer` works seems to be necessary

Reply:
yeah i'm not in love with my proposal here. I do think it is important to at least expose the encryption functionality. Another path forward here would be to modify `get_model` to optionally take a `TensorizerConfig` that would encapsulate all the tensorizer-specific stuff. It also just feels weird sprinkling all these tensorizer-specific things in the vllm APIs. Maybe it would make sense to have a separate `get_tensorized_model` function that could handle tensorizer-specific arguments?